crates/client/src/storage/fs.rs

```diff
@@ -95,11 +96,12 @@ impl RegistryStorage for FileSystemRegistryStorage {
     async fn load_packages(&self) -> Result<Vec<PackageInfo>> {
         let mut packages = Vec::new();
 
-        for entry in WalkDir::new(&self.base_dir) {
+        let packages_dir = self.base_dir.join(PACKAGE_LOGS_DIR);
+        for entry in WalkDir::new(&packages_dir) {
```

Comment:
The minimal fix for `warg update` would have been

```suggestion
        for entry in WalkDir::new(&packages_dir.join("sha256")) {
```

but given how packages are iterated over this seemed more robust.